### PR TITLE
Feature/ui update

### PR DESCRIPTION
**Pull request by chongjunkim**
# Please ensure that all information is filled out to streamline the approval of your pull request.

## Pre-Release Link
- https://github.com/cirept/salesforceToolbar/tree/v4.8.3

## Functionality Added?
- New UI, positioned within the page.
- Added "cdk-" in front of the webnum so it copies along with the actual webnum.

## Update README.md

1. [ n/a ] **README.md** - add instructions on how to use your new functionality, if applicable. ```/docs/```

### Production Release?

1. [ o ] **meta.js & meta.user.js** - Updated all instance of the version numbers in both files. ```preRelease branch -> /assets/js/```

<br>OR<br>

### Pre-Release Release?

2. [ n/a ] **preRelease.user.js** - Updated all instance of the version numbers in both files. ```develop branch -> /assets/js/```

## Update Change Log

3. [ n/a ] **Changelog.md** - add bullet points on what was added. ```/docs/```

### CODE REVIEW

**Comment by cirept** [comment]
- The userscript meta tag has the wrong version number.

**Review by cirept — CHANGES_REQUESTED** [review]
- userscript is leading to the wrong version number.  Unable to test.  =[

**Review by chongjunkim — COMMENTED** [review]
Fixed version number.

File: launchToolbar/js/launchToolbar_meta.user.js [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+// @version 4.8.3
 // @run-at document-end
-// @downloadURL https://raw.githubusercontent.com/cirept/salesforceToolbar/master/launchToolbar/js/launchToolbar_meta.user.js
+// @downloadURL https://github.com/cirept/salesforceToolbar/raw/develop/launchToolbar/js/launchToolbar_meta.user.js
 // @require https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js
 // @resource fontAwe https://use.fontawesome.com/3953f47d82.js
 // @resource googFont https://fonts.googleapis.com/css?family=Raleway:600,900
 // @resource jqueryUI https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css
-// @require https://raw.githubusercontent.com/cirept/salesforceToolbar/v4.8.2/launchToolbar\js\launchToolbar.min.js
+// @require https://github.com/cirept/salesforceToolbar/raw/develop/launchToolbar/js/launchToolbar.js
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta.user.js:13]: looks good

File: launchToolbar/js/launchToolbar_meta.user.js [hunk 2]
```diff
@@ -3,14 +3,14 @@
 // @namespace www.cobaltgroup.com
 // @include https://cdk.my.salesforce.com/a1E*
 // @include https://cdk--c.na57.visual.force.com/*
-// @version 4.8.2
+// @version 4.8.3
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta.user.js:6]: Please update the @name --> Salesforce Launch Toolbar - Pre-Release.
Thank you!

File: launchToolbar/js/launchToolbar_meta.user.js [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
 // @run-at document-end
-// @downloadURL https://raw.githubusercontent.com/cirept/salesforceToolbar/master/launchToolbar/js/launchToolbar_meta.user.js
+// @downloadURL https://github.com/cirept/salesforceToolbar/raw/develop/launchToolbar/js/launchToolbar_meta.user.js
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta.user.js:8]: looks good

**Review by cirept — APPROVED** [review]

**Pull request merged**